api/machine/machine.proto

```diff
@@ -791,11 +791,17 @@ message EtcdForfeitLeadershipResponse {
 message EtcdMemberListRequest {
   bool query_local = 1;
 }
-message EtcdMemberList {
+message EtcdMember {
+  uint64 id = 2;
+  string hostname = 3;
+  repeated string peer_urls = 4;
+  repeated string client_urls = 5;
+}
+message EtcdMembers { 
   common.Metadata metadata = 1;
-  repeated string members = 2;
+  repeated EtcdMember members = 2; 
```

Comment:
I think we should provide backwards compat here, i.e. include `repeated string legacy_members = 2;` and make new field use new tag; this way old talosctl will still work